internal/broker/broker.go

```diff
@@ -636,6 +631,44 @@ func (b *Broker) handleIsAuthenticated(ctx context.Context, session *session, au
 	return AuthGranted, userInfoMessage{UserInfo: authInfo.UserInfo}
 }
 
+// userNameIsAllowed checks whether the user's username is allowed to access the machine.
+func (b *Broker) userNameIsAllowed(userName string) bool {
+	// The user is allowed to login if:
```

Comment:
"login" is the noun, "log in" is the verb

```suggestion
	// The user is allowed to log in if:
```

Reply:
lol thank you for the clarification, I know the difference. Maybe I am a little dyslexic after all.

Reply:
Sorry, I didn't mean to imply that you don't know that. It's a very common mistake - in fact, we already used it incorrectly in the `broker.conf` file before (https://github.com/ubuntu/authd-oidc-brokers/pull/250).